### DISCUSSION TITLE
Missing 0A Wonderswan Color

### DISCUSSION THREAD

**Discussion by gladel** [eduardofilo/RG350_adam_image#169] [General]
Took the liberty to make Wonderswan Color logo for 0A theme with both 320x240 and 640x480 resolutions. Credit to the original creator.

[0A.zip](https://github.com/eduardofilo/RG350_adam_image/files/7908289/0A.zip)

**Comment by eduardofilo** [comment]
Great.
It would be even greater if you could make a pull request to the upstream repo.

https://github.com/fgl82/0A

**Reply by gladel** [reply]
@eduardofilo Thank you, buy by the life of me, I'm not familiar in doing that. May I ask if you are able to do it, please? It's yours.

**Reply by eduardofilo** [reply]
OK. No problem. It was also so that you would have the credit for the work.

**Reply by eduardofilo** [reply]
Done.

https://github.com/fgl82/0A/pull/8
